### PR TITLE
Fix data cleansing

### PR DESCRIPTION
**Pull request by ericluria**

### CODE REVIEW

File: data/final/currencies.json [hunk 1]
```diff
@@ -1211,7 +1210,7 @@
     "iso_4217_3": "USD",
     "number_decimals": 2,
     "symbols": {
-      "primary": "US$",
```
Comment (ericluria) [inline data/final/currencies.json]: @mbryzek this was changed automatically (underlying data `cldr-currencies`) has the primary symbol here as `$`

Do we need to override this? Want to be careful as I know we've had issues in the past with showing the correct symbol.

**Comment by ericluria** [comment]
Currently running `reference.go` fails because:

1) Some countries from the underlying data now have multiple display currencies separated by a comma. I added an override to default to the currency that exists in our reference data today.
2) Taiwan is mysteriously missing a lot of data
3) FLK (Falkland Islands) is missing a currency

Also, some new currencies that we don't support were added, so I've added them to the `unsupported` list.

File: data/cleansed/currency-symbols.json [hunk 2]
```diff
@@ -754,7 +753,7 @@
     "primary": "UGX"
   },
   "USD": {
-    "primary": "US$",
+    "primary": "$",
```
Comment (mbryzek) [inline data/cleansed/currency-symbols.json]: I think we need to manually override this one somehow

File: data/cleansed/countries.json [hunk 3]
```diff
@@ -1099,7 +1099,7 @@
     "currency": "XCD"
   },
   {
-    "name": "Saint Martin",
+    "name": "Saint Martin (French Part)",
```
Comment (mbryzek) [inline data/cleansed/countries.json]: would be nice to override country name here to remove `(French Part)`

File: data/final/currencies.json (hunk 1 above)
Comment (mbryzek) [reply data/final/currencies.json]: yeah - we need to keep `US$` - thanks for highlighting

**Pull request merged**